cypress/pageObjects/input.ts

```diff
@@ -0,0 +1,11 @@
+import { UIElement } from "./ui_elements";
```

Comment:
I'd split library files and app files. For example: base page, ui element, input and so on will be in "lib" subfolder and app page objects will be in the another folder.